.github/workflows/_openxla_pin_update_weekly.yml

```diff
@@ -0,0 +1,52 @@
+name: Weekly Code Update PR
```

Comment:
nit: remove the leading `_` underscore in the file name because this isn't a template

Reply:
Thanks, updated